pkgs/swiftgen/swift2objc/lib/src/parser/parser.dart

```diff
@@ -0,0 +1,23 @@
+import '_core/type_defs.dart';
+import 'parsers/ast_parser.dart';
+import 'parsers/relations_parser.dart';
+import 'parsers/symbolgraph_json_parser.dart';
+import 'parsers/symbolgraph_json_reader.dart';
+
+class Parser {
```

Comment:
Why is this a class rather than a function? I think pretty much every class in the `parsers/` directory could be a standalone function.

Reply:
Yeah, I'd prefer functions. Less boilerplate and more readable imo.

Reply:
done